test/struct_test.go

```diff
@@ -0,0 +1,133 @@
+package test
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	fuzz "github.com/google/gofuzz"
+	jsoniter "github.com/json-iterator/go"
+	"github.com/mailru/easyjson"
+	jwriter "github.com/mailru/easyjson/jwriter"
+	"github.com/philpearl/plenc"
+	plnull "github.com/philpearl/plenc/null"
+	"github.com/unravelin/null"
+)
+
+var fuzzFuncs = []interface{}{
+	func(a *null.Bool, c fuzz.Continue) {
+		a.Valid = c.RandBool()
+		if a.Valid {
+			a.Bool = c.RandBool()
+		}
+	},
+	func(a *null.Float, c fuzz.Continue) {
+		a.Valid = c.RandBool()
+		if a.Valid {
+			c.Fuzz(&a.Float64)
+		}
+	},
+	func(a *null.Int, c fuzz.Continue) {
+		a.Valid = c.RandBool()
+		if a.Valid {
+			c.Fuzz(&a.Int64)
+		}
+	},
+	func(a *null.String, c fuzz.Continue) {
+		a.Valid = c.RandBool()
+		if a.Valid {
+			c.Fuzz(&a.String)
+			if a.String == "" {
+				a.Valid = false // sigh
```

Comment:
Upstream has a major version bump for this to work as expected :sweat_smile:

Reply:
Do you think we dare make the same change? I'd quite like to...